test/suites-ostree/update.robot

```diff
@@ -0,0 +1,48 @@
+*** Settings ***
```

Comment:
========================
===== HARD HAT AREA ====
========================

Work is focused on this file now that build-automation is behaving like I want it to. So, expect it to change from what you see here.  Will remove this comment after the dust clears.

Reply:
Part of the goal of having the images, suites, and scenarios all decoupled is that we can reuse pieces to make new scenarios. This scenario needs a new image, but should be able to reuse the existing healthy upgrade RF suite, as @pmtk points out above.